app/views/idv/sessions/index.html.slim

```diff
@@ -1,51 +1,52 @@
 - title t('idv.titles.welcome')
 
-
-= form_tag(idv_sessions_path, method: 'post')
+= form_tag(idv_sessions_path, method: 'post', novalidate: true)
   h1.heading = t('idv.titles.session.basic')
   .mb3
-    label = t('idv.form.first_name')
-    = block_text_field_tag 'first_name', nil
-    label = t('idv.form.last_name')
-    = block_text_field_tag 'last_name', nil
+    = label_tag 'first_name', t('idv.form.first_name')
+    = block_text_field_tag 'first_name', nil, required: true
+    = label_tag 'last_name', t('idv.form.last_name')
+    = block_text_field_tag 'last_name', nil, required: true
   h1.heading = t('idv.titles.session.dob')
   .mb3
-    label.hide = t('idv.form.dob')
-    = block_date_field_tag 'dob', nil
+    = label_tag 'dob', t('idv.form.dob'), class: 'hide'
+    = block_date_field_tag 'dob', nil, required: true
   h1.heading = t('idv.titles.session.ssn')
   .mb3
-    label.hide = t('idv.form.ssn')
-    = block_text_field_tag 'ssn', nil
+    = label_tag 'ssn', t('idv.form.ssn'), class: 'hide'
+    = block_text_field_tag 'ssn', nil, required: true, pattern: '[0-9]*', \
```

Comment:
we should limit the SSN to 9 digits, no? With a pattern of `[0-9]{9}`? (also true for zip and whatever the number of digits for bank routing is)